config.json

```diff
@@ -3,6 +3,9 @@
     "command_channels": [
       1192895746672951340
     ],
+    "ta_review_channel": [
+      1284224818698260490
+    ],
```

Comment:
We need the TA review channel to be server specific.

Reply:
Okay I'm going to delete that part and leave it empty for the user to insert in their server